### PR TITLE
Create UI main play activity

### PR DESCRIPTION
**Pull request by ngochieu1297**
## [Pull request title]
Create UI main play activity
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7530
#### Screenshots (nếu có)
53577935_797740290590819_2113575899105329152_n

### CODE REVIEW

**Comment by ngochieu1297** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -14,6 +14,7 @@
         android:roundIcon="@drawable/ic_start"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".ui.main_play.MainPlayActivity"></activity>
```
Comment (NguyenMinhTuanB) [inline app/src/main/AndroidManifest.xml]: convention 3.1

File: app/src/main/res/layout/activity_main_play.xml [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/color_splash_screen"
+    tools:context=".ui.main_play.MainPlayActivity">
+
+    <ImageView
+        android:id="@+id/image_background"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main_play.xml]:
```suggestion
        android:id="@+id/image_back_ground"
```

File: app/src/main/res/layout/activity_main_play.xml [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:scaleType="centerCrop"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <View
+        android:layout_width="match_parent"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main_play.xml]: not match_parent

File: app/src/main/res/layout/activity_main_play.xml [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+        android:layout_height="match_parent"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main_play.xml]: tuowng tự

File: app/src/main/res/layout/activity_main_play.xml [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/dp_8"
+        android:src="@drawable/ic_not_loop_white"
+        app:layout_constraintBottom_toBottomOf="@id/image_shuffle"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/image_next" />
+
+    <android.support.constraint.Guideline
+        android:id="@+id/guideline_left"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main_play.xml]:
```suggestion
        android:id="@+id/guide_line_left"
```

Comment (NguyenMinhTuanB) [reply app/src/main/res/layout/activity_main_play.xml]: tuowng tự ở view khác

**Comment by ngochieu1297** [comment]
ready

**Pull request merged**